templates/home.macros.j2

```diff
@@ -0,0 +1,27 @@
+{% import 'content.macros.j2' as content %}
+
+
+{% macro home_cta(
+  page,
+  slug
+) %}
+  {% set page = page|get_config(slug) %}
+  {% set splash = page[0] if page else none %}
+
+  {% if splash %}
+    <section class="home-cta">
+      <h1 class="home-title">{{ splash.big|typogrify|safe }}</h1>
+      <div class="home-description">
+        {{ splash.small|rst }}
+      </div>
+      <div class="home-action">
+        {{ content.link_button(
+          content=splash.action,
+          url=splash.target
+          ) }}
```

Comment:
Very minor, but I really prefer indenting the closing `) }}` with the opening `{{`. I think it leads to clearer overall indentation and readability, and fewer following indentation errors.